packages/hop-node/src/watchers/SyncWatcher.ts

```diff
@@ -418,7 +431,6 @@ class SyncWatcher extends BaseWatcher {
         transferRootHash,
```

Comment:
Please see comment about pure `getTransferRootId()` before reading the rest of this comment.

---

We have an n-chain call here that was missed and should be removed. Ideally, it is just replaced with the pure function call.

Reply:
updated